src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs

```diff
@@ -833,13 +1001,13 @@ async Task Core(MultiConnectSocketAsyncEventArgs internalArgs, Task<IPAddress[]>
                     // Store the results.
                     if (caughtException != null)
                     {
-                        SetResults(caughtException, 0, SocketFlags.None);
+                         SetResults(caughtException, 0, SocketFlags.None);
```

Comment:
Nit: there's a space too much
```suggestion
                        SetResults(caughtException, 0, SocketFlags.None);
```